dev/com.ibm.ws.artifact.zip/resources/com/ibm/ws/artifact/zip/internal/resources/ZipArtifactMessages.nlsprops

```diff
@@ -1,3 +1,10 @@
+[ant:validate]   (error) reaper.unregistered.path is duplicate key
```

Comment:
This heading seems problematic.... doesn't seem these errors should be here, and could cause problems processing the file.

Reply:
Agh, a stray paste while adding the new warnings.  That header is not supposed to be there.